src/safe_apps/serializers.py

```diff
@@ -30,21 +32,33 @@ class NoRestrictionsAccessControlPolicySerializer(serializers.Serializer[SafeApp
     )
 
 
-class TagSerializer(serializers.Serializer[Tag]):
-    def to_representation(self, instance: Tag) -> str:
+class TagSerializer(serializers.ModelSerializer[Tag]):
+    class Meta:
+        fields = "__all__"
```

Comment:
Similar to the above:

```python
fields = ['name']
```

Reply:
Fixed in 4b2a288431b8ec3ab414b4c7f2e7a99eedc8e547.